.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "xrt"]
 	path = xrt
-	url = https://github.com/Xilinx/XRT.git
+	url = https://github.com/kashif/XRT.git
```

Comment:
Pointing XRT to your own local branch? How am I supposed to support other platforms?